content/pt/docs/concepts/security/overview.md

```diff
@@ -0,0 +1,153 @@
+Use container runtime with stronger isolation | Select [container runtime classes](/docs/concepts/containers/runtime-class/) that provider stronger isolation
+
+## Código
+
+O código da aplicação é uma das principais superfícies de ataque sobre a qual você tem maior controle.
+Embora a proteção do código do aplicativo esteja fora do tópico de segurança do Kubernetes, aqui
+são recomendações para proteger o código do aplicativo:
+
+### Segurança de código
+
+{{< table caption="Code security" >}}
+
+Área de Atenção para o Código | Recomendação |
+-------------------------| -------------- |
+Acesso só através de TLS | Se seu código precisar se comunicar por TCP, execute um handshake TLS com o cliente antecipadamente. Com exceção de alguns casos, encripte tudo em trânsito. Indo um passo adiante, é uma boa ideia encriptar o tráfego de rede entre os serviços. Isso pode ser feito por meio de um processo conhecido como mutual ou [mTLS](https://en.wikipedia.org/wiki/Mutual_authentication), que realiza uma verificação bilateral da comunicação mediante os certificados nos serviços. |
+Limitando intervalos de porta de comunicação | Essa recomendação pode ser um pouco autoexplicativa, mas, sempre que possível, você só deve expor as portas em seu serviço que são absolutamente essenciais para a comunicação ou coleta de métricas. |
+Segurança na Dependência de Terceiros | É uma boa prática verificar regularmente as bibliotecas de terceiros de sua aplicação em busca de vulnerabilidades de segurança. Cada linguagem de programação possui uma ferramenta para realizar essa verificação automaticamente. |
+Análise de Código Estático | A maioria das linguagens fornece uma maneira para analisar um extrato do código referente a quaisquer práticas de codificação potencialmente inseguras. Sempre que possível, você deve automatizar verificações usando ferramentas que podem verificar as bases de código em busca de erros de segurança comuns. Algumas das ferramentas podem ser encontradas em [OWASP Source Code Analysis Tools](https://owasp.org/www-community/Source_Code_Analysis_Tools). |
+Ataques de sondagem dinâmica | Existem algumas ferramentas automatizadas que você pode executar contra seu serviço para tentar alguns dos ataques mais conhecidos. Isso inclui injeção de SQL, CSRF e XSS. Uma das ferramentas de análise dinâmica mais populares é o [OWASP Zed Attack proxy](https://owasp.org/www-project-zap/). |
+
+{{< /table >}}
+
+## {{% heading "whatsnext" %}} 
+
+Saiba mais sobre os tópicos de segurança do Kubernetes:
+
+* [Padrões de segurança do Pod](/docs/concepts/security/pod-security-standards/)
+* [Network policies para Pods](/docs/concepts/services-networking/network-policies/)
+* [Controle de acesso à API Kubernetes](/docs/concepts/security/controlling-access)
+* [Protegendo seu cluster](/docs/tasks/administer-cluster/securing-a-cluster/)
+* [Data encryption in transit](/docs/tasks/tls/managing-tls-in-a-cluster/) for the control plane
```

Comment:
If there is no Portuguese version of a page, you can translate the title and maintain the original link.

Reply:
Hello Carol, I have been thinking about other links, which can also be translated, even if it is not possible to translate the page now.
What did you think @jailton?